driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/access.bpf.c

```diff
@@ -15,23 +15,23 @@ int BPF_PROG(access_e,
 	     struct pt_regs *regs,
 	     long id)
 {
-	struct auxiliary_map *auxmap = auxmap__get();
-	if(!auxmap)
-	{
-		return 0;
-	}
+	struct ringbuf_struct ringbuf;
+        if(!ringbuf__reserve_space(&ringbuf, ACCESS_E_SIZE))
```

Comment:
nit: @dwindsor could you check for tab vs. white space? Looking at the final diff, it seems to have mixed intend.

@Andreagit97 Question for me as reviewer Are there any guidelines regarding intend, e.g. tabs vs. white spaces?

Reply:
Just let me know what you think. Functionally this looks good to me so ready to approve from my side as well.

Reply:
Yes usually we use `tabs`, to be honest, there is an open PR to adopt a standard format https://github.com/falcosecurity/libs/pull/470 but this probably will be only merged in the next weeks :( Usually, I use clang-format-14 + the conf file in the repo (https://github.com/falcosecurity/libs/blob/master/.clang-format) to format the code.

It would be great if until we will run clang-format on the whole repo we could keep the code clean as possible, otherwise, we will fix them just in one shot when the https://github.com/falcosecurity/libs/pull/470 will be merged

Reply:
@Andreagit97 Thanks a lot for the insights.

So I guess, one more commit generated by `git diff c1d075ffda41.. |clang-format-diff -i -p1` should correct the formatting.
@dwindsor Wdyt?

Reply:
Ran clang-format on this file as well as the other new files created in this PR.